src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -274,7 +274,7 @@ def dmsetup_splitname(dev):
 ################################
 
 
-def get_api_pvs():
+def get_api_pvs(fields=None, unparsed=False, sep='";"'):
```

Comment:
Looking through the changes in this PR it is not clear to me why this function accepts customs fields, and adds the ability to have a custom separator. Where is this functionality used?

Custom fields mean it will be unable to parse (I guess that is why there is an `unparsed` flag). It also means that handling the system call has diverged and it will be tricky to normalize. Why is this needed? Why can't it be addressed with more (or different) fields? How does this help improve performance?

Reply:
I was intending to use these changes at [1] and [2]. I'll get that done...

Reply:
Updated the PR; using it now

Reply:
I see it is being used in one place now. I don't see why there is a need for a custom separator, or why does it really need to allow "unparsed" calls. How is it that this function is not being useful returning what it was returning before?